component.json

```diff
@@ -91,6 +91,41 @@
           }
         }
       }
+    },
+    "delete": {
+      "main": "./lib/actions/delete.js",
+      "title": "Delete files",
```

Comment:
Please, rename the Action to `Delete File` if it removes only one file for one execution